idunn/geocoder/models/geocodejson.py

```diff
@@ -0,0 +1,170 @@
+"""
+Implement GeocodeJson specification as defined here:
+ - https://github.com/geocoders/geocodejson-spec/tree/master/draft
+ - https://github.com/CanalTP/mimirsbrunn/blob/master/libs/bragi/src/model.rs
+"""
+from typing import List, Optional, Tuple
+
+from pydantic import BaseModel, PositiveInt, confloat
+
```

Comment:
Do we have to put an empty between each import? :)

Reply:
I usually respect this convention:

 - standard library imports
 - (empty line)
 - external libraries imports
 - (empty line)
 - project level imports

I'm not sure this is standard but most people seem to follow something close to that? I can remove this empty line if you prefer!

Reply:
This is just a nit so I don't really attach much importance to it. If the formatter is happy, then so am I.